recipes-bsp/efibootguard/efibootguard_0.13.bb

```diff
@@ -14,15 +14,15 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 SUMMARY = "A bootloader based on UEFI"
 
 SRC_URI = "gitsm://github.com/siemens/efibootguard.git;protocol=https;branch=master"
-SRCREV = "eda1b987f8ca41d64b46f53347a8c89c6fc03c31"
+SRCREV = "b6f68012fae836b470606b53aaa46fe7871ff99b"
 
 S = "${WORKDIR}/git"
 
-DEPENDS:class-target = "gnu-efi pciutils zlib libcheck"
+DEPENDS:class-target = "gnu-efi pciutils zlib libcheck autoconf-archive"
```

Comment:
Hi Adrian,
I will refactor my MR to add a single DEPENDS that work for both the target and native class.

The package autoconf-archive is needed at build time, both when compiling native and target binary, because the marco `AX_CHECK_COMPILE_FLAG` is used since https://github.com/siemens/efibootguard/commit/5fc32e98f7fcd2bc7fb7a6ea6de0f9fe214ce9ba 

It probably work because autoconf-archive is an allarch package, but you a right that it should depends on the native version of the package when building for the target too.